src/Shopsys/ShopBundle/Resources/translations/messages.de.po

```diff
@@ -2,884 +2,868 @@ msgid ""
 msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Language: en\n"
+"Language: de\n"
 
 msgid "(If I buy on the company behalf)"
-msgstr "(de) (If I buy on the company behalf)"
+msgstr "(Wenn ich im Namen des Unternehmens kaufe)"
 
 msgid "(Products will be delivered to this address)"
-msgstr "(de) (Products will be delivered to this address)"
+msgstr "(Die Produkte werden an diese Adresse geliefert)"
 
 msgid "(This address will be on the tax invoice)"
-msgstr "(de) (This address will be on the tax invoice)"
+msgstr "(Diese Adresse befindet sich auf der Steuerrechnung)"
 
 msgid "100% Satisfaction"
-msgstr "(de) 100% Satisfaction"
+msgstr "100% Zufriedenheit"
 
 msgid "24/7 Help Center"
-msgstr "(de) 24/7 Help Center"
+msgstr "24/7 Hilfe-Center"
 
 msgid "Accessories"
-msgstr "(de) Accessories"
+msgstr "Zubehör"
 
 msgid "Add company user"
-msgstr "(de) Add company user"
+msgstr "Firmenbenutzer hinzufügen"
 
 msgid "Add item"
-msgstr "(de) Add item"
+msgstr "Artikel hinzufügen"
 
 msgid "Add product"
-msgstr "(de) Add product"
+msgstr "Produkt hinzufügen"
 
 msgid "Add to cart"
-msgstr "(de) Add to cart"
+msgstr "In den Warenkorb"
 
 msgid "Added product will not have any individual discounts"
-msgstr "(de) Added product will not have any individual discounts"
+msgstr "Hinzugefügte Produkt hat keine individuellen Rabatte"
 
 msgid "Address"
-msgstr "(de) Address"
+msgstr "Adresse"
+
+msgid "All prices have to be handled manually if checked, otherwise the unit price without VAT and the total prices for that item will be recalculated automatically."
```

Comment:
This is completely new message ID (which should have been added in #53, I guess...) - how was this translation file created?

Reply:
It was created by @vitek-rostislav

Reply:
As I wrote in https://github.com/shopsys/demoshop/pull/71#discussion_r318697502. Dump is clean, this translation is in other languages as well.